src/main/java/io/cdap/plugin/http/source/batch/HttpBatchSourceConfig.java

```diff
@@ -22,8 +22,8 @@
  * Provides all the configurations required for configuring the {@link HttpBatchSource} plugin.
  */
 public class HttpBatchSourceConfig extends BaseHttpSourceConfig {
-  protected HttpBatchSourceConfig(String referenceName) {
-    super(referenceName);
+  protected HttpBatchSourceConfig(String referenceName, String authType, String oauth2Enabled) {
```

Comment:
Write a builder in the BaseHttpConfig, which is the parent of the HttpBaseSourceConfigBuilder to avoid this change.